docs/querying-data.rst

```diff
@@ -36,105 +36,105 @@ The following query returns the full JSON data for the first 3 compiled releases
 
 .. code-block:: sql
 
-  SELECT
-      data
-  FROM
-      data -- raw OCDS JSON data is stored as jsonb blobs in the `data` column of the `data` table
-  JOIN
-      compiled_release ON data.id = compiled_release.data_id -- join to the `compiled_release` table to filter data from a specific collection
-  WHERE
-      collection_id = 584
-  LIMIT 3;
+   SELECT
+       data
+   FROM
+       data -- raw OCDS JSON data is stored as jsonb blobs in the `data` column of the `data` table
+   JOIN
+       compiled_release ON data.id = compiled_release.data_id -- join to the `compiled_release` table to filter data from a specific collection
+   WHERE
+       collection_id = 584
+   LIMIT 3;
 
-To get data from a different collection, change the ``collection_id`` parameter.
+To get data from a different collection, change the ``collection_id`` condition.
 
 To get data from a collection containing releases or records, join to the ``release`` or ``record`` tables rather than the ``compiled_release`` table.
 
 .. tip:: Rendering JSON using Redash
 
-  If you are using OCP's Redash instance, you can render the results of a query as pretty printed and collapsible JSON by clicking the '+ New Visualization' button, setting the visualization type to 'table' and setting the data column to display as JSON.
+   If you are using OCP's Redash instance, you can render the results of a query as pretty printed and collapsible JSON by clicking the '+ New Visualization' button, setting the visualization type to 'table' and setting the data column to display as JSON.
 
 Calculate the total value of completed tenders in a collection
 --------------------------------------------------------------
 
-In OCDS, the tender value is stored in the ``tender.value`` `Value <https://standard.open-contracting.org/latest/en/schema/reference/#value>`__ object which consists of a numeric ``.amount`` property with an associated ``.currency``. The tender status is stored in the ``tender.status`` field.
+In OCDS, the tender value is stored in the ``tender.value`` `Value <https://standard.open-contracting.org/latest/en/schema/reference/#value>`__ object which consists of a numeric ``.amount`` field and a string ``.currency`` field. The tender status is stored in the ``tender.status`` field.
 
-To access the properties of a JSON object use the PostgreSQL ``->`` operator. The ``->`` operator takes a jsonb object and the name of a key as inputs and returns the value of the key as a jsonb value. The ``->>`` operator returns the value as a string.
+To access the properties of a JSON object use the PostgreSQL ``->`` operator. The ``->`` operator takes a JSONB object and a property's name as input, and returns the property's value as a JSONB value. The ``->>`` operator returns the value as a literal value (like text or integer).
```

Comment:
@jpmckinney I think this is wrong. `->>` always returns a string, so you need to cast the value it returns to a numeric type before you can use mathematical functions and operators on it

Reply:
Ah, yes, indeed – PostgreSQL has to do it this way since a column needs to have a single type.